tests/NTest/NTestEnv.lua

```diff
@@ -0,0 +1,60 @@
+-- A series of convenient utility functions for use with NTest on NodeMCU
+-- devices under test (DUTs).
+
+local NTE = {}
+
+function NTE.hasC(m)
+  -- this isn't great, but it's what we've got
+  local mstr = node.info('build_config').modules
+  return mstr:match("^"..m..",") -- first module in the list
+      or mstr:match(","..m.."$") -- last module in the list
+      or mstr:match(","..m..",") -- somewhere else
+      or error("Missing C module " .. m)
+end
+
+function NTE.hasL(m)
+  return pcall(require, m) or error("Missing Lua module " .. m)
```

Comment:
I would rather check for existence of a file or the module in LFS than actually requireing it.

Reply:
I'll see if I can figure out how to drive `package.loaders` myself, then.

Reply:
Ok, didn't expect other loaders to be in place. Just taking account of the 2 known ones seemed enough to me, but maybe that's not enough.